### PR TITLE
📋📷 2019 Chrysler Pacifica and 2019 Jeep Grand Cherokee support

### PR DESCRIPTION
**Pull request by adhintz**
Also likely supports arbitrary Chrysler models. This is done by copying most values from the stock LKAS camera which is active with https://github.com/commaai/panda/pull/177

2019 models only steer down to 17m/s. If LKAS is already engaged, steering down to 13m/s should be fine, but is not yet implemented.

### CODE REVIEW

File: selfdrive/car/chrysler/chryslercan.py [hunk 1]
```diff
@@ -53,28 +45,26 @@ def calc_checksum(data):
 def make_can_msg(addr, dat):
   return [addr, 0, dat, 0]
 
-def create_lkas_heartbit(car_fingerprint):
-  # LKAS_HEARTBIT (729) Lane-keeping heartbeat.
-  msg = '0000000820'.decode('hex')  # 2017
-  return make_can_msg(0x2d9, msg)
+def create_lkas_heartbit(packer, lkas_status_ok):
```
Comment (rbiasini) [inline selfdrive/car/chrysler/chryslercan.py]: I think this message can be simply forwarded in panda, rigtht?

Comment (adhintz) [reply selfdrive/car/chrysler/chryslercan.py]: Sure thing. Implemented and tested in a car that it works.

**Comment by arne182** [comment]
Thanks

**Comment by rbiasini** [comment]
I would like to merge this in. I just need the panda changes in this PR as well.

**Comment by adhintz** [comment]
Sure thing. I've now copied over the Panda change and committed it here as well.

**Pull request merged**